docs/installation-and-operations/configuration/database/README.md

```diff
@@ -19,10 +19,10 @@ point to an external database.
 Example:
 
 ```bash
-docker run -d ... -e DATABASE_URL=postgres://user:pass@host:port/dbname openproject/community:11
+docker run -d ... -e DATABASE_URL=postgres://user:pass@host:port/dbname openproject/community:12
 ```
 
-If you run the Compose based docker stack, you can simply override the `DATABASE_URL` environment variable, and remove the `db` service from the `docker-compose.yml` file. Then you can restart the stack with:
+If you run the Compose based docker stack, you can simply override the `DATABASE_URL` environment variable, and remove the `db` service from the `docker-compose.yml` file. Be careful, because by pulling a new version `docker-compose.yml` might get replaced. Best practice is using the file `docker-compose.override.yml` instead. Then you can restart the stack with:
```

Comment:
Maybe we should give advices about what to put into the `docker-compose.override.yml` to remove the `db` service without modifying `docker-compose.yml`. Personnally I do not know how to do it.

Reply:
Thank you @cbliard I also do not know about that, it is just something that @oliverguenther or @machisuji could answer